src/main/resources/xld.Folder/unpack_create.sh.ftl

```diff
@@ -13,7 +13,7 @@
 #!/bin/bash
 set -e
 
-FILE_TYPE=`file --mime-type -b ${deployed.file.path}`
+FILE_TYPE=`file -i ${deployed.file.path} | sed -n 's/.*: \(.*\);.*/\1/p'`
```

Comment:
I would like that simplification, but unfortunately the -b option isn't available to the older versions of the file command. We are hoping to end-of-life the servers where this is a problem, but in the meantime we would still need this fix to make use of this plugin on those servers.

`
file -i -b test.txt

file: illegal option -- b
usage: file [-dh] [-M mfile] [-m mfile] [-f ffile] file ...
       file [-dh] [-M mfile] [-m mfile] -f ffile
       file -i [-h] [-f ffile] file ...
       file -i [-h] -f ffile
       file -c [-d] [-M mfile] [-m mfile]
`